components/bt/host/nimble/port/include/esp_nimble_cfg.h

```diff
@@ -642,6 +642,78 @@
 #endif
 
 
+/*** @apache-mynewt-nimble/nimble/host/services/dis */
+#ifndef MYNEWT_VAL_BLE_SVC_DIS_DEFAULT_READ_PERM
+#define MYNEWT_VAL_BLE_SVC_DIS_DEFAULT_READ_PERM (-1)
```

Comment:
Currently we are on `nimble-1.2.0` release and yet to migrate to latest nimble `1.3.0` release. Few of the inclusions in latest `syscfg.h` that you have taken may need to be skipped for IDF specific NimBLE (at least for now). You may refer to [esp-nimble-dis](https://github.com/espressif/esp-nimble/blob/nimble-1.2.0-idf/nimble/host/services/dis/src/ble_svc_dis.c) for valid inclusions.

Reply:
@prasad-alatkar There was no change from `nimble-1.2.0` to `nimble-1.3.0` on `ble_svc_dis.c`, and `esp-nimble-dis` is same as the one of `nimble-1.2.0`.

Actually, the changes in nimble's `syscfg.h` are just about configuration. The file is being generated by newt's own configuration tool which is alike to menuconfig. But esp-idf doesn't use it and has no configuration tool for nimble, user has no way to change nimble configuration on his/her esp-idf project. I recommend that `esp_nimble_cfg.h` has as many inclusion as possible so that user can have a variety of options for nimble.

Reply:
Hi, sorry for delayed response. Yes, I understand that we are only adding configuration options and `nimble-1.2.0` and `nimble-1.3.0` are identical from `services/dis` point of view. I just looked for instances where `BLE_SVC_DIS_DEFAULT_READ_PERM` is used, and I could not find any. I am still ok with keeping it in `esp_nimble_cfg.h` given that it is present in upstream NimBLE (CC: @dhrishi ).